src/opds_dumper.cpp

```diff
@@ -77,6 +77,7 @@ pugi::xml_node OPDSDumper::handleBook(Book book, pugi::xml_node root_node) {
   ADD_TEXT_ENTRY(entry_node, "updated", gen_date_from_yyyy_mm_dd(book.getDate()));
   ADD_TEXT_ENTRY(entry_node, "name", book.getName());
   ADD_TEXT_ENTRY(entry_node, "flavour", book.getFlavour());
+  ADD_TEXT_ENTRY(entry_node, "category", book.getCategory());
```

Comment:
Adding the category node in the opds feed is not enough.
We have to read the node in `Book::updateFromOpds` also.
(And we need to do the same with `the library.xml` format (`Book::updateFromXml` and in `libxml_dumper.cpp`)

Reply:
I've just seen this comment : https://github.com/kiwix/kiwix-tools/issues/318#issuecomment-790691534

I'm not particularly agree with it, but let's not implement it (library.xml change) for now (until we agree in the issue discussion).

Reply:
That is contrary to @kelson42's [reply](https://github.com/kiwix/kiwix-tools/issues/318#issuecomment-790691534) to [my question](https://github.com/kiwix/kiwix-tools/issues/318#issuecomment-790680130)

Reply:
We need at least to read the node from opds.
If we don't this is totally useless to add the category in the opds feed.